hack/tools.go

```diff
@@ -23,6 +23,9 @@ import (
 	_ "knative.dev/pkg/configmap/hash-gen"
 	_ "knative.dev/pkg/hack"
 
+	// Hack scripts from eventing
+	_ "knative.dev/eventing/hack"
```

Comment:
Ahhh now I see why the vendor dir

Reply:
For what specifically we use the `eventing/hack` except the `create-kind-cluster.sh`?

Reply:
I think it's the only script we use here.

Reply:
Is it worth vendoring it with all that new deps more than just copy-pasting it?

Reply:
Ping on this @pierDipi. My worry is that `eventing/hack` is bringing in that mako proto code, which might at some point create problems with our protobuf version.

Reply:
Good point, I'll look at this.

Reply:
/hold